osu.Game/Online/API/IAPIProvider.cs

```diff
@@ -24,10 +24,15 @@ public interface IAPIProvider
         /// </summary>
         IBindableList<APIRelation> Friends { get; }
 
+        /// <summary>
+        /// The current user's status.
+        /// </summary>
+        Bindable<UserStatus> Status { get; }
```

Comment:
I guess the other thing I'd be interested in here - do you see this staying here going forward, or is this a transitory place? I have a vague unsubstantiated feeling that `IAPIProvider` having all of these pieces of state for the ambient logged-in user is not correct either and at some future point this should be the responsibility of Something Else, but I'm also aware this would probably take a 50-file refactor to execute. So I guess take this comment as a discussion point rather than something that would need immediate action.

Reply:
I see this remaining either here or directly in the `OsuConfigManager`, but not in elsewhere like `APIUser`. Right now it's just an intuitive place to expose since it's side-by-side with `Activity`.

Reply:
Yeah okay. Probably fine until it's not.